src/core/commandRunner/CommandRunner.ts

```diff
@@ -60,6 +61,10 @@ export default class CommandRunner {
       if (this.graph.debug.active) {
         this.graph.debug.log(`command:`);
         this.graph.debug.log(JSON.stringify(command, null, 3));
+        if (vscode.window.activeTextEditor) {
+          vscode.window.activeTextEditor.options.tabSize =
+            DEFAULT_TAB_SIZE_FOR_TESTS;
+        }
```

Comment:
hmm maybe we should only do this if they're recording a test?  Otherwise we'll clobber their tab size if they have cursorless debug mode on

Reply:
Heyo @pokey — I moved this entire step into the `TestCaseRecorder` since we have state there about whether we should or should not set the tabs. That also bring the code out of the `CommandRunner` loop. We clean up at the end and reset to the original tab setting, if a user stops recording their test.